### PR TITLE
fourbyte: updates

### PR DESCRIPTION
**Pull request by holiman**
New signatures from ethereum/lists

### CODE REVIEW

**Comment by karalabe** [comment]
Hmmmm.... this is a 12MB diff. Wondering whether we should replace the json with a formatted one and disable compression on the binary version. Otherwise every tiny addition we make will bring along 12MB worth of repo size, and we'll run out of allowance (not to mention it's annoying that we're already at 500MB repo size).

**Comment by holiman** [comment]
I'm sorry, I don't understand what you mean with "formatted one and disable compression". Elaborate?

**Comment by holiman** [comment]
Yes, that would also make it humanly possible to actually meaningfully review the PRs

**Comment by holiman** [comment]
This is now fixed, so the json file is sorted by key and not compressed. This time the diff will be big, but later diffs will be smaller.

File: signer/fourbyte/fourbyte.go [hunk 1]
```diff
@@ -14,7 +14,7 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
 
-//go:generate go-bindata -nometadata -o 4byte.go -pkg fourbyte 4byte.json
+//go:generate go-bindata -nometadata -nocompress -o 4byte.go -pkg fourbyte 4byte.json
```
Comment (karalabe) [inline signer/fourbyte/fourbyte.go]: Add this after the go generate to disable the linters on the dump:

```
//go:generate sh -c "sed 's#var __4byteJson#//nolint:misspell\\\n&#' assets.go > 4byte.go.tmp && mv 4byte.go.tmp 4byte.go"
```

Comment (holiman) [reply signer/fourbyte/fourbyte.go]: done

**Comment by holiman** [comment]
Travis errror on one of the jobs is unrelated

```
 --- FAIL: TestClientNotificationStorm (2.29s)
    client_test.go:378: didn't get expected error 
```

**Pull request merged**